### DISCUSSION TITLE
Support for larger screens

### DISCUSSION THREAD

**Discussion by jwestes** [RudolphRiedel/FT800-FT813#75] [General]
I have an opportunity to pick up a Crystalfontz 7" EVE display as an engineering unit.  How difficult would it be to support this larger display with a 1024 x 600 resolution?  Would it simply be a new config with different timings and resolutions or is there more to it than that?

https://www.crystalfontz.com/product/cfaf1024600b0070sca1-1024x600-7-inch-eve-tft

**Comment by RudolphRiedel** [comment]
Looks like something went wrong at some point when updating EVE_config.h as it already should include a set of timing parameters for this display.
I will put it back in this afternoon.

**Comment by jwestes** [comment]
I picked up the larger screen today and the latest code is working.  Thanks!
